com.engine.Acestream.json

```diff
@@ -0,0 +1,144 @@
+            "shared-modules/python2.7/python-2.7.json",
+            "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        {
+            "name": "pango",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/pango/1.38/pango-1.38.1.tar.xz",
+                    "sha256": "1320569f6c6d75d6b66172b2d28e59c56ee864ee9df202b76799c4506a214eb7"
+                }
+            ]
+        },       
+        {
+            "name": "apsw",
+            "buildsystem": "simple",
+            "build-commands": [
+                "touch .separate_namespace",
+                "python setup.py install --prefix=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/b5/a1/3de5a2d35fc34939672f4e1bd7d68cca359a31b76926f00d95f434c63aaa/apsw-3.9.2-r1.tar.gz",
+                    "sha256": "dab96fd164dde9e59f7f27228291498217fa0e74048e2c08c7059d7e39589270"
+                }
+            ]
+        },
+        {
+            "name": "pygtk",
+            "config-opts": ["--disable-docs"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/pygtk/2.24/pygtk-2.24.0.tar.gz",
+                    "sha256": "6e3e54fa6e65a69ac60bd58cb2e60a57f3346ac52efe995f3d10b6c38c972fd8"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "cp -p /usr/share/automake-*/config.{sub,guess} ."
+                    ]
+                }
+            ],
+            "modules": [
+                {
+                    "name": "python2-pycairo",
+                    "buildsystem": "meson",
+                    "config-opts": ["-Dpython=python2"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/pygobject/pycairo/releases/download/v1.18.1/pycairo-1.18.1.tar.gz",
+                            "sha256": "70172e58b6bad7572a3518c26729b074acdde15e6fee6cbab6d3528ad552b786"
+                        }
+                    ]
+                },
+                {
+                    "name": "pygobject",
+                    "config-opts": ["--disable-glibtest"],
+                    "cleanup": ["/bin"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.gnome.org/sources/pygobject/2.28/pygobject-2.28.7.tar.xz",
+                            "sha256": "bb9d25a3442ca7511385a7c01b057492095c263784ef31231ffe589d83a96a5a"
+                        }
+                    ]
+                }
+            ]
+        },
+            "libs/libappindicator/libappindicator-gtk2-12.10.json",   
```

Comment:
Because this should be used from shared-modules as well.

Reply:
if I use libappindicator from shared-modules, then my project simply will not start gui part (the project uses python-appindicator). So it is missing the path to libappindicator-no-python.patch Because of this, it is displayed in a separate module at the very end (after pygtk, pygobject, etc.)